test_user_management.py

```
@@ -0,0 +1,391 @@
+            return []
+    
+    def validate_password(self, password: str) -> bool:
+        """Validate password strength"""
+        # SECURITY ISSUE 9: Weak password validation
+        if len(password) >= 6:  # Too weak minimum length
+            return True
+        return False
+    
+    def hash_password(self, password: str) -> str:
+        """Hash password using MD5 (SECURITY ISSUE 10: Weak hashing)"""
+        return hashlib.md5(password.encode()).hexdigest()
```

Ruff (0.15.6)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)